module/go.mod

```diff
@@ -3,67 +3,80 @@ module github.com/Gravity-Bridge/Gravity-Bridge/module
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.9
-	github.com/cosmos/ibc-go/v3 v3.2.1
-	github.com/ethereum/go-ethereum v1.10.16
-	github.com/evmos/ethermint v0.17.0
+	github.com/cosmos/cosmos-sdk v0.45.14
+	github.com/cosmos/ibc-go/v3 v3.4.0
+	github.com/ethereum/go-ethereum v1.10.19
+	github.com/evmos/ethermint v0.21.0
 	github.com/gogo/protobuf v1.3.3
 	github.com/golang/protobuf v1.5.2
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
-	github.com/osmosis-labs/bech32-ibc v0.3.0-rc1
+	github.com/osmosis-labs/bech32-ibc v0.0.0-00010101000000-000000000000
```

Comment:
This version was updated by go automatically, it is not relevant to us given the replace directive below.

Reply:
can we just delete the line?

Reply:
Done